extensions/wikia/AnalyticsEngine/AnalyticsProviderBillTheLizard.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class AnalyticsProviderBillTheLizard implements iAnalyticsProvider {
+	private static $template = 'extensions/wikia/AnalyticsEngine/templates/bill-the-lizard.mustache';
+
+	public static function isEnabled() {
+		global $wgShowAds;
+
+		return $wgShowAds && AdEngine2Service::areAdsShowableOnPage();
```

Comment:
What do you think of including here `$noExternals`?

Reply:
It should work with $noExternals too :)
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2PageTypeService.class.php#L42